src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -141,10 +141,7 @@ const RenderListFeed = (props) => {
 
   const topicBottomPosition = () => {
     if (hasComment) {
-      if (getTotalReaction(item) === 1) {
-        return isBlurred && getHeightReaction() + normalize(4);
-      }
-      return getHeightReaction();
+      return getHeightReaction() + normalize(4);
     }
     return getHeightFooter();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of `topicBottomPosition` function simplifies the logic by always returning the height of reactions plus a normalized value if there are comments. However, it seems to ignore the condition where `getTotalReaction(item) === 1` and `isBlurred` is true. If this condition is important for your application, you might want to reintroduce it.
```diff
   const topicBottomPosition = () => {
     if (hasComment) {
+      if (getTotalReaction(item) === 1 && isBlurred) {
+        return getHeightReaction() + normalize(4);
+      }
-      return getHeightReaction() + normalize(4);
+      return getHeightReaction();
     }
     return getHeightFooter();
   };
```
If the removed condition is not necessary anymore, then the new code looks good.


<!-- This is an auto-generated comment by OSS CodeRabbit -->